vignettes/jaod.Rmd

```diff
@@ -0,0 +1,99 @@
+---
+title: "Directory of Open Access Journals"
+author: "Lluís Revilla Sancho"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Directory of Open Access Journals}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Vignettes are long form documentation commonly included in packages. Because they are part of the distribution of the package, they need to be as compact as possible. The `html_vignette` output type provides a custom style sheet (and tweaks some options) to ensure that the resulting html is as small as possible. The `html_vignette` format:
```

Comment:
Can you remove this paragraph and bulleted list?  We usually don't introduce the vignette format in vignettes.

Reply:
Oh, sorry I forgot to remove it
```suggestion
```